doc/io.rst

```diff
@@ -661,6 +661,34 @@ exporting your objects to pandas and using its broad range of `IO tools`_.
 
 .. _IO tools: http://pandas.pydata.org/pandas-docs/stable/io.html
 
+.. _io.PseudoNetCDF:
+
+
+Formats supported by PseudoNetCDF
+---------------------------------
+
+xarray can also read CAMx, BPCH, ARL PACKED BIT, and many other file
+formats supported by PseudoNetCDF, if PseudoNetCDF is installed. 
```

Comment:
Please make the first reference to "PseudoNetCDF" a link.

Reply:
Sure